### ISSUE TITLE
Version 1.4.0 not compiling on Windows 11

### ISSUE THREAD

**Issue by omarabdul-hadi** [ccouzens/leptonica-plumbing#10]
I am using this as part of the tesseract crate dependencies.
It runs great on Linux and macos.
However, on Windows it doesn't compile.
I am using vcpkg, and I have installed leptonica.
I suspect leptonica-sys version 0.4.7, is not exposing some structures that leptonica-plumbing is using.
Can someone please help me? If I figure things out, I will post any update on this thread.
I get the compile errors below:

```
error[E0432]: unresolved imports `leptonica_sys::boxCreateValid`, `leptonica_sys::boxDestroy`, `leptonica_sys::boxGetGeometry`
 --> C:\Users\Omar\.cargo\registry\src\index.crates.io-6f17d22bba15001f\leptonica-plumbing-1.4.0\src\box.rs:3:21
  |
3 | use leptonica_sys::{boxCreateValid, boxDestroy, boxGetGeometry, l_int32, l_ok};
  |                     ^^^^^^^^^^^^^^  ^^^^^^^^^^  ^^^^^^^^^^^^^^ no `boxGetGeometry` in the root
  |                     |               |
  |                     |               no `boxDestroy` in the root
  |                     no `boxCreateValid` in the root
```

**Comment by omarabdul-hadi** [comment]
Looks like the bindings.rs output file of leptonica sys is missing pretty much all the extern functions on windows. On linux the bindings file is about 5x larger. Not sure why this is occurring. This is now looking more like a problem with leptonica sys.

**Comment by ccouzens** [comment]
Hey, I'll be busy for the next week, and then should be able to be more involved. In the meantime, I should be able to review and merge PRs and push new versions.

**Comment by omarabdul-hadi** [comment]
I found the fix, and it works on windows now.
We simply need to remove the following from the build.rs script in leptonica-sys crate:

```
    #[cfg(windows)]
    {
        bindings = bindings.clang_args(["-x", "c++"]);
    }
```

I was unable to create a branch or a pull request, perhaps because I don't have access rights to create branches?
Even after fixing this, we need to spin new releases of some or all of the following to use the update: leptonica-sys, leptonica-plumbing, tesseract-sys, tesseract-plumbing, tesseract-rs

**Comment by omarabdul-hadi** [comment]
After performing local checkouts and updates to all the above crates. I was successfully able to use the tesseract crate with no issues.

**Comment by omarabdul-hadi** [comment]
Just pushed the fix, and made a pull request: https://github.com/ccouzens/leptonica-sys/pull/28

**Comment by ccouzens** [comment]
Thank you for your contribution

I've released it as [leptonica-sys 0.4.8](https://crates.io/crates/leptonica-sys/0.4.8).

The dependencies should be flexible enough on all the related projects to get this new version, but may need to delete Cargo.lock in order to get the new version of leptonica-sys.

Let me knew if you have any further issues or problems.

**Comment by omarabdul-hadi** [comment]
You are welcome :-)
You are right about the dependencies. I just built it and all is well.
